0110/streams/index.html

```diff
@@ -111,23 +93,41 @@ <h3 class="stream__text">Streams API use cases</h3>
            </div>
            <div class="customer__grid">
              <div class="customer__item streams_logo_grid streams__ny__grid">
-               <a href="https://www.nytimes.com" target="_blank" class="grid__logo__link">
+               <a href="https://medium.com/@Pinterest_Engineering/using-kafka-streams-api-for-predictive-budgeting-9f58d206c996" target="_blank" class="grid__logo__link">
```

Comment:
Hmm.. why we need to ref links for pinterest, with one for its logo and the content as well while for others we just use one ref link for the logo only?

Reply:
@guozhangwang I have link for all logos as well as link in the content.